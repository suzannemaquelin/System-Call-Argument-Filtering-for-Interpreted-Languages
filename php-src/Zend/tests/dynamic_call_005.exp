Warning: Cannot call extract() dynamically in %s on line %d
int(1)

Warning: Cannot call extract() dynamically in %s on line %d
int(1)

Warning: Cannot call extract() dynamically in %s on line %d
int(1)