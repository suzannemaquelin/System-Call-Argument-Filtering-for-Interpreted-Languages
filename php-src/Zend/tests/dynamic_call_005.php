<?php

function test_calls($func) {
    $i = 1;

    array_map($func, [['i' => new stdClass]]);
    var_dump($i);

    $func(['i' => new stdClass]);
    var_dump($i);

    call_user_func($func, ['i' => new stdClass]);
    var_dump($i);
}
test_calls('extract');

?>
