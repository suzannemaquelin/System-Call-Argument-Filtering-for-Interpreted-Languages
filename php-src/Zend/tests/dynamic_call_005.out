Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory

Warning: Cannot call extract() dynamically in /root/php-src/Zend/tests/dynamic_call_005.php on line 6
int(1)

Warning: Cannot call extract() dynamically in /root/php-src/Zend/tests/dynamic_call_005.php on line 9
int(1)

Warning: Cannot call extract() dynamically in /root/php-src/Zend/tests/dynamic_call_005.php on line 12
int(1)